File: CODE/CODE/ReverseString.java
import java.util.Arrays;

public class Solution {

    public void reverseString(char[] s) {
        int low = 0;
        int high = s.length - 1;

        while (low <= high) {
            char temp = s[low];
            s[low] = s[high];
            s[high] = temp;

            low++;
            high--;
        }
    }

    public static void main(String[] args) {

        char[] s = {'h', 'e', 'l', 'l', 'o'};

        Solution obj = new Solution();
        obj.reverseString(s);

        System.out.println(Arrays.toString(s));
    }
}

File: DSA 6th-13th June_2301220100060/Day5/reverseString.cpp
#include<iostream>
#include<vector>
using namespace std;
//Problem 344
class Solution {
public:
    void reverseString(vector<char>& s) {
        int st = 0, end = s.size()-1;

        while(st < end){
            swap(s[st++], s[end--]);
        }
        
    }
};